sources/packages/api/src/route-controllers/education-program-offering/education-program-offering.controller.ts

```diff
@@ -258,20 +260,21 @@ export class EducationProgramOfferingController extends BaseController {
   @AllowAuthorizedParty(AuthorizedParties.institution)
   @HasLocationAccess("locationId")
   @Get(
-    "location/:locationId/education-program/:programId/program-year/:programYearId/offerings-list",
+    "location/:locationId/education-program/:programId/program-year/:programYearId/selectedOffering/:selectedOffering/offerings-list",
```

Comment:
I couldn't understand what is selectedOffering here. Also it is not added to the @param in comment.

Reply:
It is offering intensity right?

Reply:
It is offering intensity selected by the student. The aim is to ensure that the list of offering intensities returned in the drop down match only what the student selected. E.g if the student selected full time, then only full time offerings should be displayed. Same for part time.

Reply:
I got the intention but as mentioned in my recent comment, selectedOffering can be renamed at API side. Both URI and variable name.